packages/gestalt/src/Pog.js

```diff
@@ -89,7 +89,11 @@ type Props = {|
 /**
  * [Pog](https://gestalt.pinterest.systems/pog) is a lower-level functional component to show the active, hovered, & focused states of [IconButton](https://gestalt.pinterest.systems/pog/iconbutton).
  *
- *This abstraction to allow for links that look like IconButton.
+ * This abstraction to allow for links that look like IconButton.
```

Comment:
(pre-existing) typo: "This _is an_ abstraction to allow for…"

Reply:
DONE